packages/fluentui/docs/src/components/GuidesNavigationFooter.tsx

```diff
@@ -19,9 +19,25 @@ const GuidesNavigationFooter: React.FC<GuidesNavigationFooterProps> = ({ previou
     <br />
     <Flex gap="gap.small">
       {previous && (
-        <Button as={Link} content={previous.name} icon="arrow left" iconPosition="before" primary to={previous.url} />
+        <Button
+          as={Link}
+          content={previous.name}
+          icon={<Icon name="arrow left" />}
+          iconPosition="before"
+          primary
+          to={previous.url}
+        />
+      )}
+      {next && (
+        <Button
+          as={Link}
+          content={next.name}
+          icon={<Icon name="arrow right" />}
```

Comment:
Should it be `< ArrowRightIcon /> `?

Reply:
This is a font icon, we don't have mapping for an svg icon

Reply:
You mean when using icon in the button is font Icon or the `arrow left` itself? I got confused because `PopupExamplePointing.shorthand.tsx` it was using `<ArrowRightIcon />`

Reply:
It is a bit confusion yeah. Basically `name="arrow-left"` can be transform to `<ArrowLeftIcon />, but here the name is `arrow left` which is a different font icon. Anyway, as it is not in an example we can change it to `<ArrowRightIcon />`, but my point was that it is a different icon.